script/github-actions/daily-product-scan/products/index.js

```diff
@@ -1,19 +1,16 @@
-const fs = require('fs');
-
 const Product = require('./product');
 
 class Products {
   constructor() {
     this.all = {};
   }
 
-  addProducts({ manifestPaths }) {
-    manifestPaths.forEach(path => {
-      const manifest = JSON.parse(fs.readFileSync(path));
-      const { productId } = manifest;
+  addProducts({ productPaths }) {
+    productPaths.forEach(path => {
+      const productId = path.product_id;
 
       if (productId) {
-        const pathToCode = path.slice(0, path.lastIndexOf('/'));
+        const pathToCode = path.path_to_code;
```

Comment:
If you grab `pathToCode` when you're destructing `path` (see my previous comment) you can delete this line:

`const pathToCode = path.path_to_code;`